client/components/PreviewImage/PreviewImage.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
This was mostly moved out of `PubPreviewImage` and given a slightly more generalized API.